uikit/viewmodels/src/main/java/com/sygic/maps/uikit/viewmodels/common/utils/UnitFormatUtils.kt

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2019 Sygic a.s. All rights reserved.
+ *
+ * This project is licensed under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.uikit.viewmodels.common.utils
+
+import com.sygic.maps.uikit.viewmodels.common.regional.units.DistanceUnit
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.*
+import kotlin.math.roundToInt
+
+object Distance {
+
+    private const val METRIC_KM_DISTANCE_UNIT = "km"
+    private const val METRIC_M_DISTANCE_UNIT = "m"
+    private const val IMPERIALS_MI_DISTANCE_UNIT = "mi"
+    private const val IMPERIALS_FEET_DISTANCE_UNIT = "ft"
+    private const val IMPERIALS_YARD_DISTANCE_UNIT = "yd"
+    private const val M_TO_KM_CONVERSION_RATIO = 0.001f
+    private const val M_TO_MI_CONVERSION_RATIO = 0.00062f
+    private const val M_TO_FT_CONVERSION_RATIO = 3.28084f
+    private const val M_TO_YD_CONVERSION_RATIO = 1.09361f
+
+    private val DECIMAL_FORMAT = DecimalFormat("#.#", DecimalFormatSymbols(Locale.ENGLISH))
```

Comment:
chceme vzdy formatovat `ENGLISH`? Podla mna asi nie. Na slovensku by to asi malo byt `4,5 km` namiesto `4.5 km`